### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by daniel-beck**
https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/

@batmat @orrc @oleg-nenashev WDYT?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- [ ] Make sure the file is created in `permissions/` directory
+- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
+- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
+- [ ] Check that the file is named `plugin-${artifactId}.yml` for plugins
+
+### When adding new uploaders (this includes newly created permissions files)
+
+- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
+- [ ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
```
Comment (batmat) [inline .github/PULL_REQUEST_TEMPLATE]: Add something like 

> make especially sure there is no typo. A way to do so is to check `https://issues.jenkins-ci.org/secure/ViewProfile.jspa?name=the_user_name` actually leads to user profile you want

?

**Comment by batmat** [comment]
@daniel-beck I'm practicing in #140 :-)

**Comment by daniel-beck** [comment]
@batmat We should to do something about inapplicable checkboxes. It looks like 4/9 now…

Any ideas?

**Comment by daniel-beck** [comment]
Merging for now to get _something_ up, can always be optimized.

**Pull request merged**